interfaces/builtin/browser_support.go

```diff
@@ -110,10 +110,25 @@ const browserSupportConnectedPlugAppArmorWithSandbox = `
 # Leaks installed applications
 # TODO: should this be somewhere else?
 /etc/mailcap r,
+# Snaps should be using xdg-open from the runtime instead of reading these
+# files directly since the snap is unable to do anything with these files
+# but these accesses have been allowed for a long time and remain so as not
+# to break existing snaps. These could be changed to explicit deny rules,
+# but that provides a worse debugging experience. Combined, the risks
+# outweigh the benefits of closing this information leak for the small
+# number of snaps allowed to use allow-sandbox: true. Reference:
+# https://forum.snapcraft.io/t/cannot-open-pdf-attachment-with-thunderbird/11845
 /usr/share/applications/{,*} r,
 /var/lib/snapd/desktop/applications/{,*} r,
 owner @{PROC}/@{pid}/fd/[0-9]* w,
 
+# Clearing the PG_Referenced and ACCESSED/YOUNG bits provides a method to
+# measure approximately how much memory a process is using via /proc/self/smaps
+# (man 5 proc). This access allows the snap to clear references for pids from
+# other snaps and the system, so it is limited to snaps that specify:
+# allow-sandbox: true.
+owner @{PROC}/@{pid}/clear_refs w,
```

Comment:
Those rule was already added by https://github.com/snapcore/snapd/commit/af48f7420c7c65f4cdb98a3e48c601bad9f3f97c . You can see it [below in the same file](https://github.com/snapcore/snapd/blob/dc9c7a2076e20ec572a708165cfcd9a035d59f5f/interfaces/builtin/browser_support.go#L250). As author of this commit I don't mind if you replace it with yours which has more extensive explanation.

Reply:
good catch, agreed either remove the other one or add this comment to the one below

Reply:
Ah, fixed. I updated this PR to modify the existing comment.